cypress/integration/1-getting-started/Local tests/ONE/ONE_2_Search_bar_functionality.spec.js

```diff
@@ -0,0 +1,32 @@
+/// <reference types="cypress" />
+const baseURL = Cypress.env('baseURL')
+// Before running this test first run the Server test 2 in order to create a user so you can log in.
+it('should be able to visit the login page', () => {
+    cy.visit(baseURL + '/') // Localhost in the env file should be changed to reflect the ONE host
+})
+
+it('should be able to log in successfully', () => {
+    cy.get('[data-test-id="input-email"]').type("cypress@test.com") // Here write your email address for your account
+    cy.get('[data-test-id="input-password"]').type("cypress123") // Write the password for your account
+    cy.get('[data-test-id="button-login-and-remember"]').click() // Here we click on the log in and remember me button
+    cy.visit(baseURL + '/') // Visiting main page of ONE
+})
+
+it('should be able to test the search bar functionality', () => {
+    cy.get('.modal-header>:last-child()').click() // Here we close the start tour pop up
+    cy.get('[data-test-id="input-search"]').type("admin") // We search for the Admin app
+    cy.wait(1000) // With this command we wait 1s before the execution of the next command. We use this for better visibility
+    cy.get('[data-test-id="input-search"]').clear().type("CRM") // We search for the CRM app 
+    cy.wait(1000)
+    cy.get('[data-test-id="input-search"]').clear().type("low code") // We search for the Low code app 
+    cy.wait(1000)
```

Comment:
how do you check here if the app appears among the other apps? Do you do it visually?